apps/hash-ai-worker-ts/src/activities/get-ai-assistant-account-id-activity.ts

```diff
@@ -23,57 +21,10 @@ export const getAiAssistantAccountIdActivity = async (params: {
   }
 
   if (grantCreatePermissionForWeb) {
-    const aiAssistantHasPermission = await graphApiClient
-      .checkWebPermission(
-        aiAssistantAccountId,
-        grantCreatePermissionForWeb,
-        "update_entity",
-      )
-      .then((resp) => resp.data.has_permission);
-
-    if (!aiAssistantHasPermission) {
-      /** The AI Assistant does not have permission in the requested web, use the web-scoped bot to grant it */
-      const webMachineActorId = await getWebMachineId(
-        { graphApi: graphApiClient },
-        authentication,
-        { webId: grantCreatePermissionForWeb },
-      ).then((maybeMachineId) => {
-        if (!maybeMachineId) {
-          throw new Error(
-            `Failed to get web machine for web ID: ${grantCreatePermissionForWeb}`,
-          );
-        }
-        return maybeMachineId;
-      });
-
-      await graphApiClient.modifyWebAuthorizationRelationships(
-        webMachineActorId,
-        [
-          {
-            operation: "create",
-            resource: grantCreatePermissionForWeb,
-            relationAndSubject: {
-              subject: {
-                kind: "account",
-                subjectId: aiAssistantAccountId,
-              },
-              relation: "entityCreator",
-            },
-          },
-          {
-            operation: "create",
-            resource: grantCreatePermissionForWeb,
-            relationAndSubject: {
-              subject: {
-                kind: "account",
-                subjectId: aiAssistantAccountId,
-              },
-              relation: "entityEditor",
-            },
-          },
-        ],
-      );
-    }
+    await addActorGroupMember(graphApiClient, authentication, {
+      actorId: aiAssistantAccountId,
+      actorGroupId: grantCreatePermissionForWeb,
+    });
```

Comment:
Posted a longer message in Slack, but the implication of this is that normal members cannot initiate AI flows (at least the first one, because only admins can add members to groups, and it needs to happen the first time an AI flow is run for a web)